### ISSUE TITLE
Multiple Logins

### ISSUE THREAD

**Issue by GoogleCodeExporter** [halalaninitiative/halalan#44]
```
What steps will reproduce the problem?
1. User A logs in with credentials C.
2. User A does not vote yet.
3. User B logs in with credentials C.
4. User B votes and denies possibility of voting for User A.
5. Wawa User A.

What is the expected output? What do you see instead?


What version of the product are you using? On what operating system?


Please provide any additional information below.


```

Original issue reported on code.google.com by `radei...@gmail.com` on 3 Mar 2008 at 5:21

**Comment by GoogleCodeExporter** [comment]
```
How can User B log in with the same credentials as User A?
```

Original comment by `djclue...@gmail.com` on 4 Mar 2008 at 7:48

**Comment by GoogleCodeExporter** [comment]
```
User B somehow see User A's username and password.

Anyway, this can be done by checking first if there is a timestamp in login 
field.
```

Original comment by `waldemar...@gmail.com` on 4 Mar 2008 at 8:03

**Comment by GoogleCodeExporter** [comment]
```
What if somebody logged in, then the browser crashed or was closed 
unexpectedly? Will
the logout field be updated?
```

Original comment by `djclue...@gmail.com` on 4 Mar 2008 at 11:09

**Comment by GoogleCodeExporter** [comment]
```
The logout field is only updated when clicking the logout field or in the 
automatic
logout after voting.

Maybe we can save the IP addresses too for checking.
```

Original comment by `waldemar...@gmail.com` on 5 Mar 2008 at 12:15

**Comment by GoogleCodeExporter** [comment]
```
Initial fix committed in r417. Please review/verify.
```

Original comment by `djclue...@gmail.com` on 7 Mar 2008 at 7:19
- Changed state: **Fixed**
- Added labels: **Priority-High**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
BTW, I didn't update the PostgreSQL db dump.
```

Original comment by `djclue...@gmail.com` on 7 Mar 2008 at 7:22

**Comment by GoogleCodeExporter** [comment]
Original comment by `toni...@gmail.com` on 10 Mar 2008 at 10:58
- Changed state: **Verified**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision be4b02fb1e.
```

Original comment by `daru...@gmail.com` on 4 Sep 2010 at 1:54
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `waldemar...@gmail.com` on 7 Feb 2011 at 9:15
- Changed state: **Verified**
